packages/core/test/financial-templates/perpetual-multiparty/PerpetualPositionManager.js

```diff
@@ -1249,7 +1498,7 @@ contract("PerpetualPositionManager", function(accounts) {
 
     // Create new oracle, replace it in the finder, and push a different price to it.
     const newMockOracle = await MockOracle.new(finder.address, timer.address);
```

Comment:
Should we test precision loss on the multiplier in another PR?

Reply:
ye, I think another PR. the precision loss I've seen in writing these tests is truncation at the 18th decimal in the cumulative funding rate, as expected.

Reply:
Ah so you've actually induced precision loss?